src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleAssociationsController.cs

```diff
@@ -55,13 +55,13 @@ public async Task<ActionResult> UpdateAssociations([FromBody] ProductAssociation
         /// Deletes specified associations
         /// </summary>
         /// <remarks>Updates the specified association</remarks>
-        /// <param name="ids">associations to delete ids</param>
+        /// <param name="id">association id to delete</param>
         [HttpDelete]
-        [Route("")]
+        [Route("{id}")]
         [Authorize(ModuleConstants.Security.Permissions.Delete)]
-        public async Task<ActionResult> Delete([FromBody]string[] ids)
+        public async Task<ActionResult> Delete(string id)
```

Comment:
There were the changes in this place in the dev, please rebase to the latest.
Also I think it is not needed to change the API - we should be able to have an ability to delete collection at once.

Reply:
fixed